quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/ls/client/ExtendedClientCapabilities.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.quarkus.ls.client;
+
+/**
+ * Extended client capabilities not defined by the LSP.
+ * 
+ * @author Angelo ZERR
+ */
+public class ExtendedClientCapabilities {
+
+	private ExtendedClientSettingsUpdateCapabilities clientSettingsUpdate;
+
+	public ExtendedClientSettingsUpdateCapabilities getClientSettingsUpdate() {
+		return clientSettingsUpdate;
```

Comment:
I think `configuration `should be used instead of `settings` (ex: ConfigurationItemEdit). A suggestion is to rename `clientSettingsUpdate` to `configurationUpdate`  or `configurationEdit`

@fbricon what do you think ?

Reply:
+1 to use configuration